### PR TITLE
auto build status page for migrations

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: migrator.js [hunk 1]
```diff
@@ -217,33 +217,39 @@ function createMigratorContainer(migratorName, parentId){
...
-                gridScale: 5,
-                gridColor: "#eeeeee",
-                data: migratorData,
-                colors: ['#440154', '#31688e', '#35b779', '#fde725']
-            });
-            migratorBarchart.draw();
-            migratorListing(migratorData, feedstockStatus, canvasData.migratorContainerId);
-        }
-    );
+function TotalMigration (migrators_dict_text, bla) {
```
Comment (scopatz) [inline migrator.js]: function names are usually camelCase in JS, I think

Comment (scopatz) [reply migrator.js]: Also, what is `bla`?

Comment (CJ-Wright) [reply migrator.js]: Place holder, I didn't want to rewrite your JSON parser, so I needed an arg to match the signature.

Comment (scopatz) [reply migrator.js]: Yeah, could you name it something more descriptive, like `placeholder`?

File: migrator.js [hunk 2]
```diff
@@ -217,33 +217,39 @@ function createMigratorContainer(migratorName, parentId){
...
+                    colors: ['#440154', '#31688e', '#35b779', '#fde725']
+                });
+                migratorBarchart.draw();
+                migratorListing(migratorData, feedstockStatus, canvasData.migratorContainerId);
+            }
+        );
+    }
 }
+var url = "https://raw.githubusercontent.com/regro/cf-graph3/master/status/total_status.json"
+loadJSON(url, "empty", TotalMigration)
```
Comment (scopatz) [inline migrator.js]: Doesn't this line need to be `totalMigration`?

**Comment by CJ-Wright** [comment]
@scopatz I think this is ready

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**